sleap/io/cameras.py

```diff
@@ -883,13 +943,13 @@ def update_points(
             if not isinstance(instance, PredictedInstance):
                 instance_oks = compute_oks(
                     gt_points[cam_idx, :, :],
-                    points[cam_idx, :, :],
+                    points_3d[cam_idx, :, :],
                 )
                 oks_scores[cam_idx] = instance_oks
 
             # Update the points for the instance
             instance.update_points(
-                points=points[cam_idx, :, :], exclude_complete=exclude_complete
+                points=points_3d[cam_idx, :, :], exclude_complete=exclude_complete
```

Comment:
Here we want to use `points_reprojected`
```suggestion
                points=points_reprojected[cam_idx, :, :], exclude_complete=exclude_complete
```